base/file.jl

```diff
@@ -461,6 +569,29 @@ end
     readdir(dir::AbstractString=".") -> Vector{String}
 
 Return the files and directories in the directory `dir` (or the current working directory if not given).
+
+# Examples
+```jldoctest
+julia> readdir(joinpath(Sys.BINDIR, Base.DATAROOTDIR))
```

Comment:
Same here, better with an example without `Sys.BINDIR` and `Base.DATAROOTDIR` IMO.